[EDU-3677] feat: metrics data transferred (#736)
aziontech/docs: src/content/docs/en/pages/main-menu/reference/build/edge-application/application-acceleration.mdx

<Button href="/en/documentation/products/edge-application/cache-settings/#caching-http-methods" text="Learn more about caching HTTP methods"></Button>

---

## Billing

**Application Acceleration** has specific metrics that define how the product is billed.

<Button href="https://www.azion.com/en/pricing/#ApplicationAcceleration" text="see pricing"></Button>

### Data Transferred

Total volume of data transferred by Azion Edge Computing Platform with the Application Acceleration module enabled, whether it's delivered through cache or fetched from an origin, calculated in GB.

> If an edge application transfers data and it has the Application Acceleration module enabled in it, charges will apply for the pricing of the module.

Reviewer: ```suggestion
If an edge application transfers data and has the **Application Acceleration** module enabled, charges will apply to the pricing of the module.
```
